package.json

```diff
@@ -22,6 +22,8 @@
   "dependencies": {
     "@headlessui/vue": "^1.4.2",
     "@heroicons/vue": "^1.0.5",
+    "@tailwindcss/forms": "^0.5.0",
```

Comment:
please remove all commits from your branch that are not relevant to your current issue.

Reply:
okay